helpers/schema.ts

```diff
@@ -0,0 +1,20 @@
+import type {Schema} from '../schema';
+import {ValueType} from '../schema';
+
+/**
+ * A helper to extract properties fromKeys from a schema object. This is mostly useful
+ * in processing the context.sync.schema in a sync formula, where the schema would only
+ * include a subset of properties which were manually selected by the Pack user.
+ */
+export function getSelectedFromKeysFromSchema(schema: Schema): string[] | undefined {
```

Comment:
I think there is room to improve the name of this helper function. "Selected" isn't accurate here, since it works for any schema, even one that hasn't be altered by the user. Also "FromKeys" isn't completely accurate, since it returns a value for properties that don't have `fromKey` set.

What about `getPropertyKeysFromSchema`?

Reply:
SGTM.

Reply:
Did you want to rename this as Eric suggested?

Reply:
just realized that I have not renamed it yet.